browser/extensions/brave_wallet_navigation_throttle.cc

```diff
@@ -47,17 +48,19 @@ BraveWalletNavigationThrottle::WillStartRequest() {
       url.host() == ethereum_remote_client_host) {
     // If a user has explicitly disabled the Brave Wallet,
     // then don't defer and try to install it.
-    if (!Profile::FromBrowserContext(web_contents->GetBrowserContext())->
-        GetPrefs()->GetBoolean(kBraveWalletEnabled)) {
+    content::BrowserContext* browser_context =
+        web_contents->GetBrowserContext();
+    Profile* profile = Profile::FromBrowserContext(browser_context);
+    if (!profile->GetPrefs()->GetBoolean(kBraveWalletEnabled) ||
+        brave::IsTorProfile(profile)) {
       return content::NavigationThrottle::BLOCK_REQUEST;
     }
-    auto* registry = ExtensionRegistry::Get(web_contents->GetBrowserContext());
+    auto* registry = ExtensionRegistry::Get(browser_context);
```

Comment:
Does this need a null check?

Reply:
I don't think so, there's a similar call in `chrome/browser/extensions/api/chrome_extensions_api_client.cc` without a null check.

Reply:
OK, gotcha